_config.yml

```diff
@@ -111,10 +111,16 @@ css_inline: false # Will insert all styles into a single <style> block in the <h
 navigation_header:
 - title: Home
   url: /
-- title: Code
-  url: /pages/code/
 - title: Documentation
   url: https://jump.dev/JuMP.jl/stable/
+- title: Community
+  items:
+  - title: Forum
+    url: https://discourse.julialang.org/c/domain/opt
+  - title: Chatroom
```

Comment:
This is a developer chatroom. We shouldn't be funneling users into it by advertising it as the JuMP chatroom.